go.mod

```diff
@@ -39,6 +39,8 @@ require (
 
 replace (
 	github.com/metal3-io/baremetal-operator => github.com/openshift/baremetal-operator v0.0.0-20200715132148-0f91f62a41fe // Use OpenShift fork
+	github.com/openshift/api => github.com/openshift/api v0.0.0-20200901182017-7ac89ba6b971
```

Comment:
why?

Reply:
because of the bump in assisted-service dependency

Reply:
@filanov please review the dependency and comment if necessary